integration/test/performancetest/performance_test.go

```diff
@@ -1,59 +1,158 @@
-// go:build linux && integration
+//go:build linux && integration
 // +build linux,integration
 
 package performancetest
 
-import(
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"sync"
 	"testing"
 	"time"
-	"log"
-	"context"
+
 	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 const (
-	configPath = "resources/config.json"
 	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
-	agentRuntimeMinutes = 5 //20min desired but 5mins for testing purposes 
+	agentRuntimeMinutes = 5 //20 mins desired but 5 mins for testing purposes
 	DynamoDBDataBase = "CWAPerformanceMetrics"
-
 )
 
 func TestPerformance(t *testing.T) {
+	configFiles := []string {
```

Comment:
```suggestion
	testcases := []struct{
	    testName             string
            tps                         int
	    agentConfig         string
	}
```
since the test case is small and also provides me a centralize place to know regarding the different test cases.

Reply:
This is getting changed in my next commit so it will be different